Server/prisma/schema.prisma

```diff
@@ -63,6 +63,17 @@ model Organization {
     contactInformation String
 }
 
+model Tickets {
+    id         Int      @id @default(autoincrement())
+    content    String
+    title      String
+    creatorId  String
```

Comment:
You don't link the creator ID with a user ? (Same for the Assigned)

Reply:
This conversation has been done privately